### PR TITLE
Add Gravitino LlamaIndex demo

### PR DESCRIPTION
**Pull request by jerryshao**
This PR proposes to add Gravitino LlamaIndex demo.

### CODE REVIEW

**Comment by jerryshao** [comment]
@qqqttt123 @xunliu please help to review.

**Comment by jerqi** [comment]
Could you add some document?

File: init/mysql/init.sql [hunk 1]
```diff
@@ -18,3 +18,12 @@
  */
 GRANT ALL PRIVILEGES on *.* to 'mysql'@'%';
 FLUSH PRIVILEGES;
+CREATE DATABASE IF NOT EXISTS `demo`;
```
Comment (jerqi) [inline init/mysql/init.sql:21]: Could you give a more meaningful name here?

File: README.md [hunk 2]
```diff
@@ -224,6 +224,35 @@ union
...
+non-tabular data with one natural language query.
+
+The demo is located in the `jupyter` folder, you can open the `gravitino_llama_index_demo.ipynb`
+demo via Jupyter Notebook by [http://localhost:8888](http://localhost:8888).
+
+The scenario of this demo is that basic structured city statistics data is stored in MySQL, and
+detailed city introductions are stored in PDF files. The user wants to know the answers to the
+cities both in the structured data and the PDF files.
+
+In this demo, we will use Gravitino to manage the MySQL table using relational catalog, pdf
```
Comment (jerqi) [inline README.md:241]: `we will` -> `you can`

File: init/mysql/init.sql (hunk 1 above)
Comment (jerqi) [reply init/mysql/init.sql:21]: How about `demo` -> `llmaindex`.

**Review by jerqi — APPROVED** [review]
LGTM. just some nits.

**Review by jerqi — APPROVED** [review]

**Review by xunliu — APPROVED** [review]
LGTM

**Comment by jerqi** [comment]
Could you create a pull request for repo of gravitino to sync the document?

**Comment by jerryshao** [comment]
Currently it cannot work with Gravitino 0.6.0, I shipped a customized 0.5.0.dev version to make it work. When Gravitino 0.6.0 is ready, I will add to that document.

**Pull request merged**